.gitignore

```diff
@@ -1,3 +1,7 @@
 .DS_Store
 node_modules/
 out/
+
+# Editor files
+.vscode
+*.code-workspace
```

Comment:
I’m not too precious about the `gitignore` file, but I’m just curious, isn‘t that only created if the repo has workspaces? 🤔

Reply:
That is correct. Again mather of habbit. Feel free to remove it.

Reply:
But workspace files would only be created in the “root” workspace repo nah? Never set one up 😄